### PR TITLE
[#OSF-7777]Clicking "Add" button multiple times when adding an unconfirmed user results in error	

### PR DESCRIPTION
**Pull request by chennan47**
## Purpose
Disable the add button in add contributor modal after it is clicked.

## QA notes
1. create a new project
2. go to the contributor page
3. add a unregistered contributor, fill in the name and click on the add button
Before change:
When you click the add button, if you are fast enough, you can click the button multiple times and get an error
After change:
when you click the add button, it is disabled.
screen shot 2017-04-13 at 11 52 21 am

## Ticket

https://openscience.atlassian.net/browse/OSF-7777

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Flake

File: website/static/js/contribAdder.js [hunk 1]
```diff
@@ -337,6 +337,8 @@ AddContributorViewModel = oop.extend(Paginator, {
     postInvite: function () {
         var self = this;
         self.inviteError('');
+        $('.btn-add').attr('disabled','disabled');
```
Comment (sloria) [inline website/static/js/contribAdder.js]: Drive-by: can you do this in Knockout instead of jQuery?

**Comment by chennan47** [comment]
@sloria Changed to Knockout.
